modules/spring-config/src/main/java/org/mule/config/spring/MuleConfigurationConfigurator.java

```diff
@@ -84,6 +89,22 @@ public Object getObject() throws Exception
         }
     }
 
+    private void determineDefaultProcessingStrategy(DefaultMuleConfiguration defaultConfig)
+    {
+        if (config.getDefaultProcessingStrategy() != null)
+        {
+            defaultConfig.setDefaultProcessingStrategy(config.getDefaultProcessingStrategy());
+        }
+        else
+        {
+            String processingStrategyFromSystemProperty = System.getProperty(MuleProperties.MULE_DEFAULT_PROCESSING_STRATEGY);
```

Comment:
In all other cases we read/use the system property in DefaultMuleConfiguration, any reason we can't do same in this case?